Arrow/Tests/ArrowTests/CDataTests.swift

```diff
@@ -43,7 +43,11 @@ final class CDataTests: XCTestCase {
             .addField("colTime64", type: ArrowType(ArrowType.ArrowTime64), isNullable: false)
             .addField("colTime64u", type: ArrowTypeTime64(.microseconds), isNullable: false)
             .addField("colTime64n", type: ArrowTypeTime64(.nanoseconds), isNullable: false)
-            .addField("colTime64", type: ArrowType(ArrowType.ArrowTime64), isNullable: false)
+            .addField("colTimestamp", type: ArrowType(ArrowType.ArrowTimestamp), isNullable: false)
+            .addField("colTimestamptsn", type: ArrowTypeTimestamp(.nanoseconds), isNullable: false)
+            .addField("colTimestamptsu", type: ArrowTypeTimestamp(.microseconds), isNullable: false)
+            .addField("colTimestamptsm", type: ArrowTypeTimestamp(.milliseconds), isNullable: false)
+            .addField("colTimestamptss", type: ArrowTypeTimestamp(.seconds), isNullable: false)
```

Comment:
* It seems that `s` is redundant
* Should we use `seconds` -> `milliseconds` -> `microseconds` -> `nanoseconds` order like `Time{32,64}`?

```suggestion
            .addField("colTimestampts", type: ArrowTypeTimestamp(.seconds), isNullable: false)
            .addField("colTimestamptm", type: ArrowTypeTimestamp(.milliseconds), isNullable: false)
            .addField("colTimestamptu", type: ArrowTypeTimestamp(.microseconds), isNullable: false)
            .addField("colTimestamptn", type: ArrowTypeTimestamp(.nanoseconds), isNullable: false)
```